verilog/analysis/verilog_project.h

```diff
@@ -297,6 +297,13 @@ class VerilogProject {
   const VerilogSourceFile* LookupFileOrigin(
       absl::string_view content_substring) const;
 
+  // Returns relative path to the VerilogProject
+  std::string GetRelativePathToSource(absl::string_view absolute_filepath);
+
+  // Updates file from external source, e.g. Language Server
```

Comment:
We pass the raw pointer to the file content. What are the lifetime and ownership expectations of this object ?
So this needs at least a documentation comment.

(In particular, the language server will replace the view relatively often, do we know that we never retain a stale object ?)